app/views/shared/_navbar.html.erb

```diff
@@ -26,7 +26,35 @@
           <%= link_to "Community", chat_link, class: 'nav-link', target: '_blank', rel: 'noreferrer' %>
         </li>
 
-        <li class="nav-link">
+        <% if cookies[:theme] == "dark" %>
+          <li class="nav-item nav-link" id="dark-mode-sun" style="flex-direction: row; justify-content: center; overflow:visible">
+            <%= link_to root_path(theme: "light"), data: { test_id: "dark-mode-button" }, class: "dark-mode-button sun" do %>
```

Comment:
I was a little confused when seeing the link to the root paths here. It may be worth creating a new "Themes" controller we could reference instead? It wouldn't need much, just an update action that would hold the logic from the `set_theme` method currently on the application controller.

I think that would reveal the intention for this better and provide a place for this feature to change/grow without affecting anything else.

Reply:
CC: @CouchofTomato

Reply:
Yeah I did mention to in the chat Tati that we might need a controller when we first discussed moving it server side. I went against it because I thought this was a temporary thing until we decided long term how we want to handle a dark mode. No issue with creating one though.